pkg/datastore/postgres/repositories/checked_headers_repository.go

```diff
@@ -17,62 +17,77 @@
 package repositories
 
 import (
+	"fmt"
+
 	"github.com/makerdao/vulcanizedb/pkg/core"
 	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
 )
 
 const (
-	insertCheckedHeaderQuery = `UPDATE public.headers SET check_count = (SELECT check_count WHERE id = $1) + 1 WHERE id = $1`
+	insertCheckedHeaderQuery = `
+INSERT INTO %s.checked_headers (check_count, header_id)
+VALUES (1, $1)
```

Comment:
We did - that's how the ON_CONFLICT WORKS.  See line 32 where there is the +1.

What has changed is that entries in the join table aren't present until the first check, whereas before we relied on the header row having a default check_count of 0. Now we try to insert a check row, and if it's present (ON CONFLICT) we increment it by one.

This should be the same behavior as before, as the only place I saw where we explicitly set a value was in tests. Otherwise it was the same  "increment by 1" logic.

Reply:
😅 I feel very roasted by you putting "works" in all caps 🔥

Reply:
😆 That was unintentional. I was just STILL TYPING THE SQL AND I GUESS KEPT GOING.